src/ExplainProvenanceImpl.h

```diff
@@ -742,6 +742,8 @@ class ExplainProvenanceImpl : public ExplainProvenance {
             return std::make_tuple(-1, -1, std::vector<RamDomain>());
         }
 
+        // TODO (darth_tytus): update to reflect new types.
+
```

Comment:
Is this done?

Reply:
This is related to float and unsigned. This fragment should be updated
and also the SouffleInterface should. But it is not strictly related
to this pull request. 
Should I update it now, or leave it with other float/unsigned related
updates (aggregates, user-defined functors...) for the time after the
records I/O?

Reply:
#1274

Reply:
Ok - let's revisit when record I/O is implemented. We should not forget.